### PR TITLE
Fix html tests, and bug clicking on date outside of month

### PR DESCRIPTION
**Pull request by lukewestby**
https://www.pivotaltracker.com/story/show/161559169

I added a comment where the bug was fixed. The rest is just test upgrade.

### CODE REVIEW

File: src/DateTimePicker.elm [hunk 1]
```diff
@@ -1256,9 +1256,20 @@ dateClickHandler pickerType stateValue year month day =
         selectedDate =
             DateTime.fromDate year month day.day
 
+        adjustedSelectedDate =
```
Comment (lukewestby) [inline src/DateTimePicker.elm:1259]: Adding this allows for the correct month to be selected

File: .gitignore [hunk 2]
```diff
@@ -2,4 +2,5 @@ elm-stuff/
 repl-temp-*
 npm-debug.log
 node_modules/
-dist/
\ No newline at end of file
+dist/
+demo/index.html
```
Comment (terezka) [inline .gitignore:6]: Didn't we just delete that one? 🤔

**Review by terezka — APPROVED** [review]
Looks great! Only one comment:

File: .gitignore (hunk 2 above)
Comment (lukewestby) [reply .gitignore:6]: I accidentally committed it to begin with. Just making sure I don't do that again.

**Pull request merged**